configure.py

```diff
@@ -1442,6 +1442,9 @@ def main():
     write_action_env_to_bazelrc('ROCM_PATH', environ_cp.get('ROCM_PATH'))
     write_action_env_to_bazelrc('ROCM_ROOT', environ_cp.get('ROCM_PATH'))
 
+  if environ_cp.get('TF_NEED_ROCM') == '1':
```

Comment:
I guess this was just for easier testing? Although we also want to flip the flag soon for cuda devices, there are still some blockers for that, for example it seems we still use the old kernels in some tensorflow codepaths even when enabling this flag, and then there can be small numeric differences in the results. But maybe this is not such a big deal, because these differences would also show up when running on different backends. We are still discussing this internally whether it is really a blocker, and what we can do about it.

Reply:
>I guess this was just for easier testing?

yes. In the long run, the expectation would be that this flag would have the same default value in ROCm and CUDA.